triggers/sg_jira_event_trigger.py

```diff
@@ -9,6 +9,16 @@
 import logging
 import requests
 import urlparse
+
+try:
+    # Allow users to define their sensible data in a .env file and
+    # load it in environment variables with python-dotenv.
+    # https://pypi.org/project/python-dotenv/
+    from dotenv import load_dotenv
+    load_dotenv(override=True)
```

Comment:
I added a note about being able to add it to the same .env file as the previous step. Let me know if you think this needs more explanation. I decided to add the dotenv method because the settings.py file is already using it. This way you can define all your variables in one place. Since it was already being used, it was already set up in the gitignore